# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/gtk-perl10/lib_versions.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.PL.orig	2003-08-05 20:00:52.000000000 +0200
+++ ./Makefile.PL	2003-08-05 20:01:29.000000000 +0200
@@ -22,9 +22,9 @@
 		check => sub {ccompile('#include <gtkgl/gtkglarea.h>', "", `gtk-config --cflags`,
 					"-lgtkgl  -lGL -lGLU " . `gtk-config --libs`)}},
 	gdkpixbuf => {order => 2, dir => 'GdkPixbuf', depends => [qw(gtk)],
-		check => sub {cconfig('gdk-pixbuf-config --version', '0\.(8|9|1\d)\.?')}},
+		check => sub {cconfig('gdk-pixbuf-config --version', '0\.(2|8|9|1\d)\.?')}},
 	gtkhtml => {order => 2, dir => 'GtkHTML', depends => [qw(gtk)],
-		check => sub {cconfig('gnome-config --modversion gtkhtml', 'gtkhtml-0\.[89]')}},
+		check => sub {cconfig('gnome-config --modversion gtkhtml', 'gtkhtml-((1\.0)|(0\.[89]))')}},
 	gtkxmhtml => {order => 2, dir => 'GtkXmHTML', depends => [qw(gtk)],
 		check => sub {cconfig('gnome-config --libs gtkxmhtml', '-lgtkxmhtml')}},
 	gnome => {order => 3, dir => 'Gnome', depends => [qw(gdkimlib)],
